gis/libpcl/Portfile

```diff
@@ -44,10 +44,9 @@ configure.args-append -DBUILD_apps=ON \
                       -DWITH_DOCS=OFF \
                       -DWITH_QT=ON
 
-variant openmp description {Build with macports-clang >= 5.0 to enable OpenMP for speed} {
-    depends_lib-append      port:libomp
-    compiler.blacklist-append *gcc* {clang}
-    compiler.blacklist-append macports-clang-3.3 macports-clang-3.4 macports-clang-3.7
+variant openmp description {Enable OpenMP for speed} {
+    # TODO: verify OpenMP version required
+    compiler.openmp_version 3.1
```

Comment:
I haven't determined exactly what version of OpenMP is required, it may be lower or higher than 3.1. I have selected version 3.1 based on the port's existing preference for at least the OpenMP functionality in clang 5.0 (version 3.1; see https://trac.macports.org/ticket/56430 and https://releases.llvm.org/5.0.0/tools/clang/docs/UsersManual.html#openmp-features), which appears to have the same level of OpenMP support as in clang 3.8 (see https://releases.llvm.org/3.8.0/tools/clang/docs/UsersManual.html#openmp-features).